lua/ulx/modules/sh/cfc_refund.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Refund"
+
+local playerKills = {}
+
+hook.Add( "PlayerInitialSpawn", "OnPlayerJoin", function(ply)
+	playerKills[ply] = {}
+end)
+
+hook.Add( "PlayerDisconnected", "OnPlayerDisconnect", function(ply)
+	playerKills[ply] = nil
+
+	for k, killData in pairs( playerKills )
```

Comment:
```suggestion
	for _, killData in pairs( playerKills )
```
use underscores for values that are never used